### PR TITLE
EDU-503: Add explanation about notifications of upcoming certificate expirations

### PR DESCRIPTION
**Pull request by djmagee**
## Preview

**https://temporal-documentation-git-edu-503-cert-notifications.preview.thundergun.io/cloud/certificates#expiration-notifications**

## What does this PR do?

Adds a section to [How to manage certificates in Temporal Cloud](https://docs.temporal.io/cloud/certificates) about notifications generated by Temporal Cloud when a CA certificate is near expiration.

### CODE REVIEW

File: docs-src/cloud/certificates-notifications.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+To keep your Namespace secure and online, you must update the CA certificate for the Namespace _before_ the certificate expires.
+
+To help you remember to do so, Temporal Cloud sends email notifications to users who have the Global Admin [Role](/cloud/users-account-level-roles) or the Namespace Admin [permission](/cloud/users-namespace-level-permissions) 15 days before expiration and, if necessary, 10 days before expiration.
+
+If the certificate is not updated, 5 days before expiration Temporal Cloud creates a [support ticket](/cloud/support-create-ticket) on behalf of the Global Admins and Namespace Admins.
+
+To ensure that you receive email notifications, configure your junk-email filters to permit email from `noreply@temporal.io`.
+
+After a support ticket is created, admins should expect a follow-up from the Temporal Developer Success team.
```
Comment (jlacefie) [inline docs-src/cloud/certificates-notifications.md:20]: based on recent feedback, may we add a line to inform users to reach out to Dev Success to opt out of or provide feedback about notifications, pls?

**Review by jlacefie — CHANGES_REQUESTED** [review]
requested 1 new line in the How to receive notifications section to provide direction for users who want to change their notification recipients or have feedback about notifications. TL;DR - until we build affordances in Temporal Cloud for Notifications, Dev Success tickets are the right avenue for Notification canges.

**Review by jlacefie — APPROVED** [review]

File: docs-src/cloud/certificates-namespace.md [hunk 2]
```diff
@@ -19,8 +19,10 @@ To manage certificates for Temporal Cloud Namespaces, use the **Namespaces** pag
 
 Don't let your certificates expire!
 Add reminders to your calendar to issue new CA certificates well before the expiration dates of the existing ones.
+Temporal Cloud begins sending notifications 15 days before expiration.
+For details, see the previous section ([How to receive notifications about certificate expiration](/cloud/certificates-notifications)).
 
-When updating CA certificates, it's important to follow a rollover process.
+When updating CA certificates, it's important to follow a rollover process (sometimes referred to as "certificatev rotation").
```
Comment (aarohib) [inline docs-src/cloud/certificates-namespace.md:25]:
```suggestion
When updating CA certificates, it's important to follow a rollover process (sometimes referred to as "certificate rotation").
```

File: docs-src/cloud/certificates-notifications.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+To help you remember to do so, Temporal Cloud sends email notifications to users who have the Global Admin [Role](/cloud/users-account-level-roles) or the Namespace Admin [permission](/cloud/users-namespace-level-permissions) 15 days before expiration and, if necessary, 10 days before expiration.
+
+If the certificate is not updated, 5 days before expiration Temporal Cloud creates a support ticket on behalf of the Global Admins and Namespace Admins.
+
+To ensure that you receive email notifications, configure your junk-email filters to permit email from `noreply@temporal.io`.
+
+After a support ticket is created, admins should expect a follow-up from the Temporal Developer Success team.
+
+To change who receives certificate notifications for a Namespace (or to provide feedback about notifications), [create a support ticket](/cloud/support-create-ticket).
```
Comment (aarohib) [inline docs-src/cloud/certificates-notifications.md:22]: Specify '...certificate expiry notifications..."?

**Review by aarohib — APPROVED** [review]

**Pull request merged**